hummingbot/connector/exchange/ndax/ndax_exchange.py

```diff
@@ -779,7 +783,7 @@ async def _update_order_status(self):
         # Initial parsing of responses. Joining all the responses
         parsed_history_resps: List[Dict[str, Any]] = []
         for resp in raw_responses:
-            if not isinstance(resp, Exception):
+            if not isinstance(resp, Exception) and resp["ClientOrderId"] in self.in_flight_orders:
```

Comment:
It might be a good idea to separate the validation of "ClientOrderId" being in our collection of in flight orders. If we receive and update for an order we are not tracking it could be confusing if we log that as an error.

Reply:
Yea I agree. Alright will make the necessary changes.

Reply:
I realise that it will simply be ignored in `_process_order_event_message`. So I will be removing this condition.